var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -39,8 +39,9 @@ def install(self, spec, prefix):
             make_defs += ['BUILD_LAPACK_DEPRECATED=1']
 
         # Add support for OpenMP
-        # Note: Make sure your compiler supports OpenMP
-        if '+openmp' in spec:
+        # Note: Apple's Clang 7.3.0 does not support OpenMP.
+        # For now, disable OpenMP if we compile with clang.
+        if ('+openmp' in spec) and (self.compiler.name != 'clang'):
             make_defs += ['USE_OPENMP=1']
```

Comment:
@davydden Please provide at least a warning : clang is not only Apple's Clang and it would be annoying  if a spec like
```
openblas+openmp%clang
```
build without complaints discarding the `openmp` support. Have you considered just changing the default value for `openmp` and exit with failure if `openblas+openmp%clang` is built on OSX?